bottles/backend/wine/winecommand.py

```diff
@@ -282,15 +282,17 @@ def get_env(self, environment: Optional[dict] = None, return_steam_env: bool = F
             env.add("__GL_SHADER_DISK_CACHE_PATH", os.path.join(bottle, "cache", "gl_shader"))
             env.add("MESA_SHADER_CACHE_DIR", os.path.join(bottle, "cache", "mesa_shader"))
 
-        # VKD£D environment variables
+        # VKD3D environment variables
         if params.vkd3d and not return_steam_env:
             env.add("VKD3D_SHADER_CACHE_PATH", os.path.join(bottle, "cache", "vkd3d_shader"))
 
         # LatencyFleX environment variables
         if params.latencyflex and not return_steam_env:
             _lf_path = ManagerUtils.get_latencyflex_path(config.LatencyFleX)
-            _lf_icd = os.path.join(_lf_path, "layer/usr/share/vulkan/implicit_layer.d/latencyflex.json")
+            _lf_layer_path = os.path.join(_lf_path, "layer/usr/share/vulkan/implicit_layer.d")
+            env.concat("VK_ADD_LAYER_PATH", _lf_layer_path)
             env.add("LFX", "1")
+            ld.append(os.path.join(_lf_path, "layer/usr/lib/x86_64-linux-gnu"))
         else:
             env.add("DISABLE_LFX", "1")
```

Comment:
So now we pull from the Bottles repos and set it all up here, instead of using what ships with the Bottles flatpak?

Reply:
In short: Yes.

In less short: LatencyFlex has configurable versions just like DXVK and VKD3D-Proton, previously we were in a mixed setup where the dll's libraries were those of the user chosen LatencyFlex version, and the native library from the one shipped with flatpak. Now it correctly uses everything from the version chosen by the user, meaning there's no more risks of an incompatible dll/native libraries couple as before, among other things. It also means we don't need to ship it with flatpak anymore.